docs/tutorial/fast_data/bucket_storage_support_tutorial.md

```diff
@@ -0,0 +1,235 @@
+---
+id: bucket_storage_support_tutorial
+title: Bucket Storage Support Tutorial
+sidebar_label: Bucket Storage Support
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import SourceCodeBlock from "./../../../src/components/SourceCodeBlock";
+
+The Bucket Storage Support enables storing messages flowing through a channel into an external storage system,
+such as a [Google Cloud Storage bucket](https://cloud.google.com/storage) or any bucket compatible with [Amazon AWS S3](https://aws.amazon.com/s3/) interface.
+  
+## What We Will Build
+
+In this tutorial, you will configure a Console Project containing the Bucket Storage application, that will store data coming from Kafka ingestion topics.
+In particular, you will:
+  - Create the microservices of the Bucket Storage Support solution;
+  - Configure ingestion-topics and store the incoming message in a sequence of batch text files in a dedicated bucket folder;
+  - Expose an endpoint to reload the data from the bucket files into a dedicated re-ingestion topic.
+
+## Prerequisites  
+- Knowledge of [Fast Data architecture](/fast_data/architecture.md) and [Bucket Storage Support](/fast_data/bucket_storage_support/overview.md).
+- An up-and-running Kafka cluster.
+- Either GCP or S3-like bucket connection.
+
+## Setup the Microservices
+
+The Bucket Storage Support can be either configured for GCP Bucket or S3 Bucket, and it provides two microservice:
+
+* _Ingestion Storer_, to configure with ingestion-topics that will be stored in the corresponding bucket folder
+* _Ingestion Reloader_, a REST microservice that will use bucket's SDK to produce message into a topic, usually referred to as **re-ingestion topic**, specified by the incoming request.
+
+You can create this microservices as plugins starting directly from the Console Marketplace, or either create an [Application](/marketplace/applications/mia_applications.md), choosing between S3-like bucket or GCP bucket.
```

Comment:
Qua non ho capito se l'application esiste oppure no. Se l'abbiamo già fatta andrebbe pubblicata sul Marketplace come tutte le altre, e quindi poi cambierei questa frase dicendo che esiste l'application. @flower-of-the-bridges sei riuscito ad aprire la pr per pubblicarla o hai bisogno di supporto?

Reply:
Ciao @TheRusCompass 

L'applicazione è stata già inserita sul marketplace, ho anche aperto la la pr